dpdbaltics.php

```diff
@@ -1127,7 +1127,7 @@ public function printLabel($idShipment)
             if ($isAutomated) {
                 if (!$shipment->return_pl_number) {
                     $shipmentService = $this->getModuleContainer('invertus.dpdbaltics.service.shipment_service');
-                    $shipmentService->createReturnServiceShipment(Config::RETURN_TEMPLATE_DEFAULT_ID, $shipment->id_order);
+                    $shipment = $shipmentService->createReturnServiceShipment(Config::RETURN_TEMPLATE_DEFAULT_ID, $shipment->id_order);
```

Comment:
The is is to reassign and use further down somewhere right?

Reply:
Yeah right after if conditions I pass it to method witch concats these two labels 
`$parcelPrintResponse = $this->printConcatedLabels($shipment, $format, $position);`